metagraph/src/common/vectors/wavelet_tree.hpp

```diff
@@ -207,7 +207,7 @@ class wavelet_tree_sdsl : public wavelet_tree {
     bool load(std::istream &in);
     void serialize(std::ostream &out) const;
 
-    void clear() { wwt_ = t_wt_sdsl(); }
+    void clear() { wwt_ = t_wt_sdsl(); count_.assign(count_.size(), 0); }
```

Comment:
so this was broken before this? do we not have any unit tests which test `clear`?

Reply:
It wasn't designed to be re-used after you call `clear()`, so there were no issues.
I just thought it's easy to make it re-usable and added this cheap change.
So now a vector can be re-used after clear().

But I have no use-cases for this right now. When we do, we should write unit tests.
For now, I'd still assume that clear() may invalidate the vector.

Reply:
So it's not a "bugfix", rather geeneralization/improvement which may be helpful in the future.